README.adoc

```diff
@@ -119,6 +123,30 @@ require'nvim-treesitter.configs'.setup {
 
 Some troubleshooting tips if the installation isn't working:
 
-* If using neovim 0.5, ensure you are using the 0.5-compat branch of the nvim-treesitter/nvim-treesitter plugin
+* Ensure you are using neovim 0.11 or higher.
 * Run `:TSInstall pkl` to manually install the Pkl parser.
 * If syntax highlighting doesn't work until you `:edit` the pkl file to reload it, ensure that the `pkl-neovim` plugin is configured to run after the `nvim-treesitter` plugin.
+
+== Configuration
+
+To configure pkl-neovim, set `vim.g.pkl_neovim`.
+See `:h pkl-neovim` for detailed documentation on all of the available configuration options.
+
+=== Setting up Pkl Language Server
+
+pkl-neovim can be enhanced with the https://pkl-lang.org/lsp/current/index.html[Pkl Language Server].
+
+To use, first download the latest LSP release from https://github.com/apple/pkl-lsp/releases[GitHub].
+
+Once downloaded, configure `vim.g.pkl_neovim.start_command`:
+
+[source,lua]
+----
+vim.g.pkl_neovim = {
+  start_command = { "java", "-jar", "path/to/pkl-lsp.jar" }
```

Comment:
Would it make sense to default this to `{"pkl-lsp"}`?

Reply:
I guess long-term it would be nice to publish said executable in the github releases and then make this default change, but fair enough in the meantime.